scipy/integrate/_ivp/ivp.py

```diff
@@ -383,7 +392,8 @@ def solve_ivp(fun, t_span, y0, method='RK45', t_eval=None, dense_output=False,
         Number of LU decompositions.
     status : int
         Reason for algorithm termination:
-
+        
+            * -2: Other exception occured during integration step, and was caught.
```

Comment:
Not sure if "other" makes sense to me here. Maybe "Exception during the integration step was caught."?

Reply:
Agreed